src/main/java/uk/gov/hmcts/darts/task/runner/impl/UnstructuredToArmAutomatedTask.java

```diff
@@ -32,8 +32,13 @@ protected void runTask() {
         unstructuredToArmProcessor.processUnstructuredToArm();
     }
 
+    @SuppressWarnings("java:S4507")
     @Override
     protected void handleException(Exception exception) {
+        if (exception instanceof NullPointerException npe) {
+            log.error("Exception with null: {}", npe.getMessage());
```

Comment:
I think you might need to add the `npe` as the last param on the log.error to see the stacktrace alongside your error message? Not sure about adding the printStackTrace bit below - unless just temporary